daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
hi @calavera  I don't find it is poplulated.  It seems volume in List() and Get() populate different